### PR TITLE
Wffhcohort-655

### PR DESCRIPTION
**Pull request by shelleyMurphy**
Adding PMD rule that calls out size > 0 calls and recommends replacing them with CollectionUtils.isEmpty

### CODE REVIEW

File: fhir-client-config/pom.xml [hunk 1]
```diff
@@ -71,5 +71,11 @@
 			<artifactId>mockito-core</artifactId>
 			<scope>test</scope>
 		</dependency>
+		<dependency>
+			<groupId>org.apache.commons</groupId>
+			<artifactId>commons-collections4</artifactId>
```
Comment (csandersdev) [inline fhir-client-config/pom.xml:76]: This is a new dependency for all of our projects. Is there any need to introduce Collections v4 or can it be just the v1 API like the rest of our projects?

Comment (shelleyMurphy) [reply fhir-client-config/pom.xml:76]: Whoops, let my IDE add the dependency. Just rerunning the tests on the other dependency now, but I think that should be fine.

**Review by csandersdev — APPROVED** [review]
LGTM

**Review by tabishop — APPROVED** [review]
LGTM

**Review by jaigoradia — APPROVED** [review]

**Pull request merged**